### PR TITLE
chore(bug): improved the ember to angular regex

### LINKED ISSUE #203: Invalid URL in version output
When I execute 'ng --version' the output contains http://www.angular-cli.com/user-guide/#watchman which does not exists.

```
$ng --version
Could not find watchman, falling back to NodeWatcher for file system events.
Visit http://www.angular-cli.com/user-guide/#watchman for more info.
node: 5.3.0
npm: 2.14.10
os: darwin x64
```

### PR DESCRIPTION
**Pull request by Brocco**
The regex was ovewriting a URL on ng version
Fixes #203

### CODE REVIEW

**Comment by cironunes** [comment]
Thanks for the PR. I think that it won't work. Just open the console and try:

```js
'ember-cli.com'.replace(/ember-cli(?!.com)/g, 'angular-cli'); // ember-cli.com
```

**Comment by jkuri** [comment]
Thanks for the fix @Brocco. I was planning to fix this but stuck with virus in bed for the last four days. I will check this later today, also `ember-cli-build.js` is being replaced in `ng new` and should be fixed.

File: lib/cli/index.js [hunk 1]
```diff
@@ -8,16 +8,16 @@ module.exports = function (options) {
       if (/version:/.test(string) || /warning:/.test(string)) {
         return;
       }
-      string = string.replace(/ember-cli/g, 'angular-cli');
-      string = string.replace(/ember/g, 'ng');
+      string = string.replace(/ember-cli(?!.com)/g, 'angular-cli');
```
Comment (jkuri) [inline lib/cli/index.js]: :+1:

**Comment by jkuri** [comment]
LGTM

**Pull request merged**